m4/ax_check_library.m4

```diff
@@ -61,35 +60,36 @@
 #   modified version of the Autoconf Macro, you may extend this special
 #   exception to the GPL to apply to your modified version as well.
 
-#serial 7
+#serial 5
```

Comment:
Is this m4 file going to a previous version?  From version 7 to version 5.  All the other files appear to be moving forward.

Reply:
This is one of the files that Brian had made his own updates to (serial 6 and serial 7) for his application. I'm assuming he's returning that file to the actual released version as his changes don't apply to our build process. (I'm am verifying my assumption with him.)